charts/opentelemetry-demo/values.yaml

```diff
@@ -612,6 +612,14 @@ opentelemetry-collector:
     opentelemetry_community_demo: "true"
   config:
     receivers:
+      filelog:
+        exclude:
+          - '/var/log/pods/*-otelcol*/opentelemetry-collector/*.log'
+          # logs are sent as OTLP, so we filter out the file logs
```

Comment:
@puckpuck this sounds like a bug with the demo.  Is the demo purposefully emitting the logs both as otlp and stdout?

Reply:
I thought this is the intended behavior - but I didn't see a definitive answer in https://opentelemetry.io/docs/specs/otel/logs/

Reply:
@zeitlinger are you adding logs to the demo yourself? I don't believe the demo does logs over otlp in 1.6

Reply:
I have enabled the flag for jvm (adservice, frauddetectionservice), but not for the others (recommendationservice/python, cartservice/dotnet, quoteservice/php)

Reply:
I would actually propose to add the java flag to enable logging as well - it will be enabled by default with 2.0 (of the java agent)

Reply:
The demo has started to add pieces to support logging, but it isn't complete yet. Any logging that services do is likely some defaults from an SDK that is not yet configured.